### PR TITLE
Vp cli docs

### PR DESCRIPTION
**Pull request by vipyne**
Fixes # 

## Proposed Changes

  - Add `docs/CLI.md`
  - Add `Table of Contents` to `README.md`
  - Add `yargs` to package.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,20 @@ One liner description of the repo.
...
+* [API](#api)
+* [CLI](#cli)
+* [Contributing](#contributing)
+* [See Also](#see-also)
+
 ## Status
 **Stable**
 
-> **Important**: Important oddities and common gotchas.
+> **Important**: Important oddities and common gotchas. Use this syntax throughout document.
```
Comment (madelinecameron) [inline README.md]: I'm confused by what you mean, maybe explain this more. :)

File: README.md [hunk 2]
```diff
@@ -23,63 +33,55 @@ More details...
...
-*EXAMPLE RESPONSE HERE*
-```
-
+### Example
 ```js
 const reponame = require('ara-reponame')
 
 const result = reponame.doStuff(32, 'ara-docs')
-console.log(result)
+`${console.log(result)}`
```
Comment (madelinecameron) [inline README.md]: Typo?

File: README.md [hunk 3]
```diff
@@ -23,63 +33,55 @@ More details...
...
+## CLI
+See [command line README][/docs/CLI.md].
 
 ## Contributing
 - [Commit message format](/.github/COMMIT_FORMAT.md)
 - [Commit message examples](/.github/COMMIT_FORMAT_EXAMPLES.md)
 - [How to contribute](/.github/CONTRIBUTING.md)
-
-Releases follow [Semantic Versioning](https://semver.org/)
+- [Release Versioning guidelines](https://semver.org/)
```
Comment (madelinecameron) [inline README.md]: The fact this is upper upper lower always bothers me. "Versioning" doesn't seem like it should capitalized

File: docs/cli.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
+# Ara Module Template Command Line Guide
```
Comment (madelinecameron) [inline docs/cli.md]: Point to cli-docs-generator script or hold off until it supports more formats? :)

File: README.md (hunk 2 above)
Comment (vipyne) [reply README.md]: so with this... I'm unsure of how to communicate that the _actual_ logged out result should be here. (not _literally_ `console.log(result)`... thoughts?

File: docs/cli.md (hunk 4 above)
Comment (vipyne) [reply docs/cli.md]: mmm... let's wait til it is used by at least one of aid, ann, act, or afs

**Pull request merged**